docs: Add screencast recording script

eadamsintel opened the pull request:
This script will create an asciinema recording to show users
how to use the PMEM-CSI operator and drivers along with a
sample application deployed from the PMEM-CSI GitHub.

Signed-off-by: Eric Adams <eric.adams@intel.com>

pohly commented on docs/screencast-pmem.sh:
```diff
@@ -0,0 +1,415 @@
+  sleep 1
+  command 'kubectl get no -o json | jq .items[].metadata.labels'
+  sleep 2
+  out '#  Nothing is yet labeled for persistent memory.'
```
```suggestion
  out '#  Nothing is labeled yet for persistent memory.'
```

pohly commented on docs/screencast-pmem.sh:
```diff
@@ -0,0 +1,415 @@
+{
+  clear
+  prompt
+  out '#  Now install the PMEM-CSI operator. This can either be done by using the yaml files on'
```
```suggestion
  out '#  Now install the PMEM-CSI operator. This can be done either by using the yaml files on'
```

pohly commented on docs/screencast-pmem.sh:
```diff
@@ -0,0 +1,415 @@
+  clear
+  prompt
+  out '#  Now install the PMEM-CSI operator. This can either be done by using the yaml files on'
+  out '#  the PMEM-CSI GitHub or by using the Operator Hub. If you are using OpenShift then the'
```
```suggestion
  out '#  the PMEM-CSI GitHub or by using the Operator Hub. If you are using OpenShift, then the'
```

pohly commented on docs/screencast-pmem.sh:
```diff
@@ -0,0 +1,415 @@
+  command 'kubectl get packagemanifest | grep pmem-csi'
+  sleep 2
+  out '#  To install an Operator you need to:'
+  out '#  1) Create pmem-csi namespace if it does not exist.'
```
```suggestion
  out '#  1) Create the "pmem-csi" namespace if it does not exist.'
```

pohly commented on docs/screencast-pmem.sh:
```diff
@@ -0,0 +1,415 @@
+  sleep 2
+  command 'kubectl apply -f https://github.com/intel/pmem-csi/raw/v1.0.1/deploy/common/pmem-storageclass-late-binding.yaml'
+  sleep 2
+  out '#  Create a Persistent Volume Claim for that storage class.'
```
```suggestion
  out '#  Create a PersistentVolumeClaim for that StorageClass.'
```

pohly commented on docs/screencast-pmem.sh:
```diff
@@ -0,0 +1,415 @@
+  sleep 2
+  command 'kubectl apply -f https://github.com/intel/pmem-csi/raw/v1.0.1/deploy/common/pmem-pvc-late-binding.yaml'
+  sleep 2
+  out '#  If you describe the PVC you will see the volume is still waiting for its first consumer.'
```
```suggestion
  out '#  If you describe the PVC, you will see the volume is still waiting for its first consumer.'
```

pohly commented on docs/screencast-pmem.sh:
```diff
@@ -0,0 +1,415 @@
+  sleep 1
+  command 'kubectl get pods'
+  sleep 2
+  out '#  Inside this pod there is an application called pmem-dax-check which can verify if'
```
Not sure about this one...

```suggestion
  out '#  Inside this pod there is an application called pmem-dax-check which can verify whether'
```

pohly commented on docs/screencast-pmem.sh:
```diff
@@ -0,0 +1,415 @@
+  command 'kubectl get pods'
+  sleep 2
+  out '#  Inside this pod there is an application called pmem-dax-check which can verify if'
+  out '#  MAP_SYNC is supported. If it is then it confirms the file being checked was accessed'
+  out '#  from a persistent memory volume.'
```
```suggestion
  out '#  MAP_SYNC is supported. If that works, then it confirms that the file is'
  out '#  stored on persistent memory.'
```

pohly reviewed (changes requested):
You should be able to merge all suggestions that you agree with via GitHub. Then locally, pull from your repo to get the changes.

pohly reviewed (changes requested):
Some short usage instruction at the top of the file would be useful. As it stands now, I have no clue what to do with it :sweat_smile:

eadamsintel commented:
I updated everything with your feedback and added instructions at the beginning on how to use. Ironically, I found after all the reviews "ouput" was spelled wrong.  I think I have everything correct now but if not can record another video.

pohly commented:
I'll update the link to the final animation separately and then backport that change to the versioned v1.0 documentation.

pohly commented:
https://asciinema.org/a/Bz0KrHGgwHMSokekheZazDR2m
